pkgs/applications/science/math/gap/default.nix

```diff
@@ -80,12 +93,18 @@ stdenv.mkDerivation rec {
   ];
 
   nativeBuildInputs = [
+    autoconf # TODO: try removing after updating
```

Comment:
Why is this needed at all?

Reply:
Because `io` fails to build otherwise. I think you fixed this in https://github.com/gap-system/gap/pull/5070 but I haven't checked to be sure.

Reply:
I just tested the aforementioned PR and unfortunately it does not fix the problem (sorry, I would have reported it upstream otherwise). Line 194 is the last line from the following rule:

```
gen/pkgconfig.h: gen/pkgconfig.h.stamp
        @if test ! -f $@; then rm -f $<; else :; fi
        @if test ! -f $@; then $(MAKE) $<; else :; fi
```

Reply:
No and it is not meant to; the patch needs to be applied to the  `io` package instead, and one other affected package (forgot which, will look it up later). A fixed release of io has already been made, too

Reply:
Ah, my bad, thanks for clarifying.